operators/api/v1alpha2/tenant_types.go

```diff
@@ -98,6 +98,14 @@ type TenantResourceQuota struct {
 	Instances uint32 `json:"instances"`
 }
 
+// TenantResourceQuota defines maximum and current resource quota for each Tenant.
+type TenantResourceQuota struct {
+	// The amount of used resources associated with this Tenant.
+	Used *TenantResourceQuotaData `json:"used,omitempty"`
+	// The amount of limits resources available to this Tenant.
+	Limits *TenantResourceQuotaData `json:"limits,omitempty"`
```

Comment:
These are not required to be pointers (pointers are typically used in APIs to distinguish whether a field has not been set or has been assigned the zero value, but in this case it is not necessary and adds a bit of complexity).